src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IVariableDeclaration.cs

```diff
@@ -1498,6 +1498,48 @@ static void Main(string[] args)
             VerifyOperationTreeAndDiagnosticsForTest<VariableDeclarationSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
 
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact, WorkItem(32100, "https://github.com/dotnet/roslyn/issues/32100")]
+        public void UsingDeclaration_OperationKind_None()
+        {
+            //TODO: implement IOperation for using declarations
+            string source = @"
+using System;
+
+class C : IDisposable
+{
+    public void Dispose()
+    {
+    }
+
+    public static void M1()
+    {
+        /*<bind>*/using var c = new C();/*</bind>*/
```

Comment:
> /*<bind>*/using var c = new C();/*</bind>*/ [](start = 7, length = 44)

Please add a test for multiple declarations. Also, it feels like this test belongs in the file dedicated to ```using``` statements